control-board/src/tasks/control_task.rs

```diff
@@ -170,34 +170,41 @@ impl <
 
             let basic_telem = TelemetryPacket::Basic(BasicTelemetry {
                 sequence_number: 0,
-                robot_revision_major: 0,
+                robot_revision_major: 2,
                 robot_revision_minor: 0,
-                battery_level: battery_voltage,
-                battery_temperature: 0.,
                 _bitfield_align_1: [],
+                battery_percent: 0,
+                kicker_charge_percent: 0,
                 _bitfield_1: BasicTelemetry::new_bitfield_1(
-                    0, 0, 0, self.shared_robot_state.ball_detected() as u32, 0, 0, 0, 0, err_fl, hall_err_fl, err_bl, hall_err_bl, err_br, hall_err_br, err_fr, hall_err_fr, err_drib, hall_err_drib, 0, 0, 0,
+                    0, 1, 0, 0, 0, // power battery error flags not supported by this generation
+                    self.shared_robot_state.shutdown_requested() as u32,
+                    self.shared_robot_state.robot_tipped() as u32,
+                    0, self.shared_robot_state.ball_detected() as u32, 
+                    self.shared_robot_state.get_imu_inop() as u32, 0, 
+                    self.shared_robot_state.get_imu_inop() as u32, 0, 
+                    err_fl, hall_err_fl,
+                    err_bl, hall_err_bl,
+                    err_br, hall_err_br,
+                    err_fr, hall_err_fr,
+                    err_drib, hall_err_drib, 
+                    self.shared_robot_state.get_kicker_inop() as u32, 0, 1,
+                    1, 0, 0,
```

Comment:
Would be good to comment these bitfield entries but if this is just for backcomp then it's probably fine

Reply:
back port only, old version from comp is tagged, and this was tested on field